src/telemetry.ts

```diff
@@ -47,7 +47,12 @@ export interface NewComponentCommandProps {
     'use_existing_devfile': boolean;
 }
 
-export type TelemetryProps = Partial<CommonCommandProps & NewComponentCommandProps>;
+export interface copyDevFileProps {
+    'devFileName': string;
+    'devFileCopied': boolean;
```

Comment:
defFileCopied property is not needed because command id itself serves the same purpose.

Reply:
That means copyDevFileProps interface is not needed at all.